### PR TITLE
Fix Various Transpiler and Update StatModule

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Review by Thundermaker300 — APPROVED** [review]

**Review by IRacle1 — APPROVED** [review]

File: Exiled.Events/Patches/Events/Player/SendingAdminChatMessage.cs [hunk 1]
```diff
@@ -54,7 +54,10 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     // message
                     new(OpCodes.Ldarg_0),
 
-                    // SearchingPickupEventArgs ev = new(Player, ItemPickupBase, SearchSession, SearchCompletor, float);
+                    // true
+                    new(OpCodes.Ldc_I4_1),
+
+                    // SearchingPickupEventArgs ev = new(Player, string, bool);
```
Comment (IRacle1) [inline Exiled.Events/Patches/Events/Player/SendingAdminChatMessage.cs:60]: oh stop why it is SearchingPickupEventArgs

**Review by IRacle1 — CHANGES_REQUESTED** [review]

File: Exiled.Events/Patches/Events/Player/SendingAdminChatMessage.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.Events/Patches/Events/Player/SendingAdminChatMessage.cs:60]: because yamato does copy past really dumbly

**Review by ALEXWARELLC — COMMENTED** [review]
Doesn't this contradict #2188

**Comment by Misaka-ZeroTwo** [comment]
Fix build error and conflict

**Comment by louis1706** [comment]
> Fix build error and conflict

Done sorry i haven't seen your comment

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**